ghost/core/core/server/api/endpoints/utils/serializers/output/utils/clean.js

```diff
@@ -68,6 +68,9 @@ const author = (attrs, frame) => {
         if (attrs.location === '') {
             attrs.location = null;
         }
+
+        // if a twitter username is provided, use it as the x username to ensure backwards compatibility
+        attrs.x = attrs.twitter || null;
```

Comment:
Although this pattern is not the greatest, the intention of the `clean.js` utility is for cleaning & removing attributes. There is an `extra-attrs.js` utility intended for adding attributes.